packages/visx-scale/src/operators/unknown.ts

```diff
@@ -11,6 +11,6 @@ export default function applyUnknown<
   config: ScaleConfigWithoutType<Output, DiscreteInput, ThresholdInput>,
 ) {
   if ('unknown' in scale && 'unknown' in config && typeof config.unknown !== 'undefined') {
-    scale.unknown(config.unknown);
+    (scale.unknown as any)(config.unknown);
```

Comment:
The latest type definitions for `d3-scales` are causing issues at here and that's why I have including the type casting in here

Reply:
Would you mind sharing the specific issue? 
Just want to help think if there is a workaround that is more type-safe than casting to `any`.

Reply:
` /**
     * Returns the current unknown value, which defaults to undefined.
     */
    unknown(): UnknownReturnType<Unknown, undefined>;
    /**
     * Sets the output value of the scale for undefined (or NaN) input values and returns this scale.
     *
     * @param value The output value of the scale for undefined (or NaN) input values.
     */
    unknown<NewUnknown>(value: NewUnknown): ScaleLinear<Range, Output, NewUnknown>;`

Above is the latest type definition from `@types/d3-scale`. Somehow the `scale.unknown` is pointing to the function which accepts no arguments.

Reply:
@kristw Please let me if any other workaround can be done to avoid this.

Reply:
It looks like TS isn't correctly matching the second overload. is there any way to import `NewUnknown` and coerce to that?

`scale.unknown(config.unknown as NewUnknown)`

Reply:
@williaster I have tried that option as well. They are not exposing the `NewUnknown` for us to import

Reply:
@kristw any other thoughts? else we can probably merge this as is